staticjinja/reloader.py

```diff
@@ -39,16 +83,45 @@ def event_handler(self, event_type, src_path):
 
         """
         filename = os.path.relpath(src_path, self.searchpath)
+        # Avoid vim super temporary file
+        if filename.endswith('4913'):
+            return
```

Comment:
It's clearly a mess but I was frustrated with tests in real situation where this caused trouble. But we can't have extra tests for all such cases anyway so I guess it's better to remove this one.